### ISSUE TITLE
[NPC] Marshfang Slicer, Marshfang Ripper, Bloodthirsty Marshfang, "Count" Ungula

### ISSUE THREAD

**Issue by netherwingbta** [Atlantiss/NetherwingBugtracker#2120]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)

**Description**: 

The following mobs should spawn with no mana and then mana drain until they can use their abilities:
https://www.wowhead.com/npc=18131/marshfang-slicer
https://www.wowhead.com/npc=18285/count-ungula
https://www.wowhead.com/npc=20196/bloodthirsty-marshfang
https://www.wowhead.com/npc=18130/marshfang-ripper

Ripper: https://youtu.be/hBOjPHXsj6c?t=13s
Marshfang Slicer: https://youtu.be/bzLYStDy7rA?t=31s
2007 vid: https://www.youtube.com/watch?v=lWO3fRAu96k
"Count" Ungula: https://youtu.be/GqvshZ9ztLc?t=5s

Currently since they start at full mana they are able to use their abilities as soon as possible and they seem to have a short cd as well.
Marshfang Slicer's nether shock is used as soon as you pull and according to this [wowhead comment](https://www.wowhead.com/npc=18131/marshfang-slicer#comments:id=57143:reply=5066) the dmg may be off. Currently does 1300 when the comment suggests ~500.

I don't know if they currently have mana drain since i'm a warrior. 
**Server Revision**: 
2244

**Comment by killablink** [comment]
I was gonna report this too but yes, same thing as he explained. He uses drain mana but he just spams nether shock which is doing way too much dmg

**Comment by Mikik3jr** [comment]
Yeah they drain mana from my paladin. Here is a TBC video proof of them having 0 mana at start:
https://www.youtube.com/watch?v=lWO3fRAu96k

**Comment by killablink** [comment]
Also good to note that they just spam too much their abilities with low cd, he barely even autoattacks because he's trying to use all his abilities

**Comment by zauru7** [comment]
Can confirm, the fact that they can spam their nether shock abilites makes it hard to kill them
